e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -1094,7 +1092,7 @@ H.describeEE("scenarios > embedding > full app", () => {
           type: "frame",
           frame: {
             mode: "fit",
-            height: Cypress.sinon.match(value => value < 400),
+            height: Cypress.sinon.match(value => value < 1000),
```

Comment:
Related to https://github.com/metabase/metabase/pull/52639/files#r1929621034.
The empty dashboard will now take more vertical space.